### PR TITLE
Deprecate layout widget 'ui-creme-container'. Obsolete since all browsers are supporting flexbox, column & table layouts

### PR DESCRIPTION
**Pull request by joehybird**

### CODE REVIEW

**Review by genglert — CHANGES_REQUESTED** [review]
I seems 'creme.layout.preferredSize' is still used (frame.js ...). And even if you do not remove it, & just deprecated it, only deprecated code should use deprecated code (i.e. vanilla code should not cause deprecation log ; only custom code should).

**Comment by joehybird** [comment]
creme.layout.preferredSize is not deprecated for now (after your remark I've added a todo). Just the resize detector.

File: CHANGELOG.txt [hunk 1]
```diff
@@ -113,6 +113,9 @@
         # In 'creme_core.templatetags.creme_bricks':
             - 'brick_action' accepts a new boolean argument 'comeback' ; once enabled it will add a GET argument
               'callback_url' with the current location (client side).
+        # Remove obsolete javascript tools : all browsers are supporting flexbox, table layout and ResizeObserver
+            - creme.widget.Container
+            - creme.layout.LayoutResizeSensor
```
Comment (genglert) [inline CHANGELOG.txt:118]: You are in the "non breaking change" section, but they are breaking changes.

**Review by genglert — CHANGES_REQUESTED** [review]

File: CHANGELOG.txt (hunk 1 above)
Comment (joehybird) [reply CHANGELOG.txt]: get lost in changelog sorry

**Review by genglert — APPROVED** [review]

**Pull request merged**